app.js

```diff
@@ -0,0 +1,78 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+var session = require('express-session');
+var FileStore = require('session-file-store')(session);
+var passport = require('passport');
+var authenticate = require('./authenticate');
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
+
+const mongoose = require('mongoose');
+const db_url = 'mongodb+srv://medhavigupta6:mongodb@cluster0-h3jmt.mongodb.net/users?retryWrites=true&w=majority';
```

Comment:
you can't keep username and password in the code directly! anyone can use your DB !!!